app/Http/Controllers/Races/MemberController.php

```diff
@@ -45,7 +43,6 @@ public function index(Campaign $campaign, Race $race)
             ->has('character.entity')
             ->leftJoin('characters as c', 'c.id', 'character_race.character_id')
             ->sort(request()->only(['o', 'k']), ['c.name' => 'asc'])
-            ->filter($filters)
```

Comment:
Why was the filter removed? We need it to filter to just the current race vs all child races